polkadb/database.go

```diff
@@ -26,8 +26,8 @@ import (
 
 // BadgerDB struct contains directory path to data and db instance
 type BadgerDB struct {
-	path string
-	db   *badger.DB
+	Datadir string
```

Comment:
Should this point directly to the provided `./chaindata` or a directory inside? looking at existing projects database will likely just be a sub-directory I think.

Reply:
Yeah if existing projects are doing such we should keep the status quo.